src/main/scala/scodec/interop/scalaz/package.scala

```diff
@@ -20,6 +21,14 @@ package object scalaz extends scalaz.ScalazInstancesLowPriority {
   implicit class AttemptSyntax[A](val self: Attempt[A]) extends AnyVal {
     /** Converts this attempt to a disjunction. */
     def toDisjunction: Err \/ A = self.fold(\/.left, \/.right)
+
+    /** Converts this attempt to a task. */
+    def toTask: Task[A] =
+      toTask(err => new IllegalArgumentException(err.message))
```

Comment:
I think that's pretty reasonable. I tend to use it everywhere else, so I'm not sure why I didn't here.